exercise_01/transformStringToCollection.js

```diff
@@ -12,3 +13,19 @@
  * output
  *  - array
  */
+
+/**
+ * Returns onError or onSuccess function depending if value is a string and if it's not empty
+ * @param {*} value
+ * @param {function} onError
+ * @param {function} onSuccess
+ * @returns {function}
+ */
+function transformStringToArray(value, onError, onSuccess) {
+  if (!validateString(value)) {
+    return onError();
+  }
+  return onSuccess(value);
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion

  return onSuccess(value);
```